get-deps

```diff
@@ -223,22 +223,10 @@ gentoo_deps() {
   portageq envvar USE | xargs -n 1 | grep '^X$' \
   || (echo 'X is not found in USE flags' && exit 1)
   EMERGE="$SUDO emerge"
-  $EMERGE --select \
-    'cmake' \
-    'fontconfig' \
-    'openssl' \
-    'git' \
-    'libX11' \
-    'libxkbcommon' \
-    'pkgconf' \
-    'python' \
-    'perl' \
-    'wayland' \
-    'mesa' \
-    'xcb-util' \
-    'xcb-util-image' \
-    'xcb-util-keysyms' \
-    'xcb-util-wm'
+  for i in 'cmake' 'fontconfig' 'openssl' 'dev-vcs/git' 'libX11' 'libxkbcommon' 'pkgconf' 'python' 'perl' 'wayland' 'mesa' 'xcb-util' 'xcb-util-image' 'xcb-util-keysyms' 'xcb-util-wm'
```

Comment:
For the sake of easier diffs in the future, please preserve the one-line-per-package format that we had before:

```
  for i in \
      'cmake' \
      'fontconfig' \
```